sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java

```diff
@@ -725,6 +773,16 @@ private Map<String, String> getDistroVersionsOptionsMap(Set<String> versions, Ve
 		return optionsMap;
 	}
 
+	private Map<String, String> getO3VersionsOptionsMap(VersionsHelper versionsHelper,
+			String optionTemplate, String artifactTemplate) {
+		Map<String, String> optionsMap = new LinkedHashMap<>();
+		Artifact artifact = new Artifact("referenceapplication-distro", "3.0.0-SNAPSHOT", "org.openmrs.distro", "zip");
```

Comment:
Will this continue to work when we move from `3.0.0-SNAPSHOT` to `3.1.0-SNAPSHOT` or any other version?

Reply:
yes, I even tested it with random strings.

Reply:
We can't use null or "" because the version cannot be empty.

Reply:
When i asked about null above, you said they all return the same.

Reply:
> Even for versions that do not exist? How about `NULL`?

> yes, I even tested it with random strings.

I answered the first part of your question and accidentally submit the comment. So I commented `We can't use null "" or because the version cannot be empty.` after that.GitHub didn't show me this when I replied . Sorry for the confusion.


> In that case, let us pass NULL such that it does not appear like we are hardcoding the current snapshot version of 3.0.0-SNAPSHOT

Reply:
Can you take a look at the implementation of that method to find out why it requires a version parameter whose value it does not respect? Or at least, instead of hardcoding version 3.0.0-SNAPSHOT, can we use the version of the distro that we are setting up?

Reply:
We use `SDKConstants.getReferenceModule("2.3.1")`. We could create something like this for O3. But still we are not avoiding the hard coding though.

https://github.com/openmrs/openmrs-sdk/blob/545ea9cff6bfaabf8bc7645a5994945155dc3c5a/sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java#L687-L695

Reply:
> We use `SDKConstants.getReferenceModule("2.3.1")`. We could create something like this for O3. But still we are not avoiding the hard coding though.
> 
> https://github.com/openmrs/openmrs-sdk/blob/545ea9cff6bfaabf8bc7645a5994945155dc3c5a/sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/DefaultWizard.java#L687-L695

https://github.com/openmrs/openmrs-sdk/blob/545ea9cff6bfaabf8bc7645a5994945155dc3c5a/sdk-commons/src/main/java/org/openmrs/maven/plugins/utility/SDKConstants.java#L109-L113

Reply:
Also, I didn't use `versionsHelper.getSuggestedVersions` because it only returns one version for O3. I think we should switch to it in the future when there are more versions of O3.

Reply:
@wikumChamith can you think of anything else that we can do to avoid hard coding the 3.0.0-SNAPSHOT version?

Reply:
We could just pass a string like `"version"`

Reply:
Of the two, which do you think is better?

Reply:
If you choose to leave the 3.0.0-SNAPSHOT version, at least put a comment for the maintainers of this code to know why you put this version.